docs/user-management.md

```diff
@@ -22,21 +22,31 @@ The architecture integrates with a separate Auth server. This Auth server is use
 
 ## General Deployment
 
-In order to deploy, there are several steps:
+User management requires the following:
 
-1. There must be an existing, preconfigured auth server for Tornjak components to integrate with. 
-2. The backend is deployed with a configuration pointing to said auth server. 
-3. The frontend must be deployed configured to obtain access tokens from said auth server before sending calls to the backend. 
+*  There must be an existing, pre-configured Auth Server to integrate with Tornjak (e.g. Keycloak)
+*  The Tornjak Backend is deployed with a configuration pointing to said Auth Server. 
+*  The Tornjak Frontend must be deployed configured to obtain access tokens from said Auth Server before sending calls to the Tornjak Backend. 
 
 ## Examples and Tutorials
 
-We have experimented extensively with the open source Keycloak Auth server. Specifically, it is very easy to set up a Tornjak secured by Keycloak today. 
+We have experimented extensively with the open source Keycloak Auth Server. 
+Specifically, it is very easy to set up a Tornjak secured by Keycloak today, by following these steps: 
 
-To configure the Auth server, please see our [Medium blog](https://medium.com/universal-workload-identity/step-by-step-guide-to-setup-keycloak-configuration-for-tornjak-dbe5c3049034) for a walkthrough on configuring the auth server. For more in-depth documentation on this setup, please see [this document on Keycloak configuration](./keycloak-configuration.md).
+1. Configure Auth Server
+To configure the Auth Server, please see our [Medium blog](https://medium.com/universal-workload-identity/step-by-step-guide-to-setup-keycloak-configuration-for-tornjak-dbe5c3049034) for a walkthrough on configuring the Auth Server. For more in-depth documentation on this setup, please see [this document on Keycloak configuration](./keycloak-configuration.md).
```

Comment:
There needs to be an extra line in between or a colon. This shows up as one line

Reply:
same thing for the other two items in the list